PRIVACY_POLICY.md

```diff
@@ -130,35 +139,37 @@ We use the information you provide to customize your experience on the site. We
 
-From time to time COS may email you electronic newsletters, announcements, surveys, or other information unrelated to any Project. If you prefer not to receive any or all of these communications, you may opt out by following the directions provided within the electronic newsletters and announcements. COS also may conduct polls and surveys of our users, and your participation in this type of research is at your sole discretion. COS may follow up with you regarding your participation in this research. You may at any time opt out of participating in these polls and surveys.
+From time to time COS may email you electronic newsletters, announcements, surveys, research studies (see section 13), or other information unrelated to any Project. If you prefer not to receive any or all of these communications, you may opt out by following the directions provided within the electronic newsletters and announcements. COS also may conduct polls and surveys of our users, and your participation in this type of research is at your sole discretion. COS may follow up with you regarding your participation in this research. You may at any time opt out of participating in these polls and surveys.
 
 Further, COS may send you occasional Service-related emails that you may not opt-out of (e.g. changes or updates to features of our Services that have security or privacy implications, technical and security notices, account verification).
 
-### 16. DATA RETENTION  
+### 17. DATA RETENTION  
 We will keep your personal information for as long as your account is active or as needed to comply with our legal obligations, even after you've closed your account, such as to meet regulatory requirements, resolve disputes between users, to prevent fraud and abuse, or to enforce this Privacy Policy and our Terms of Use. We may be required to retain personal information for a limited period of time if requested by law enforcement. We also may retain indefinitely non-personally identifiable, aggregate data to facilitate our ongoing operations.
 
-### 17. GENERAL DATA PROTECTION REGULATION  
+### 18. GENERAL DATA PROTECTION REGULATION  
 If you are a resident of or are located in the European Economic Area (“EEA”), you may have certain rights under the General Data Protection Regulation (“GDPR”).  Personal data you provide is only collected with your consent, and may be transmitted outside of the EEA to COS (or computer servers maintained for the benefit of COS) pursuant to that consent. 
 
 In general, under the GDPR you may:
```

Comment:
Needs a newline to properly bullet the following list, when we render on OSF pages
```suggestion
In general, under the GDPR you may:

```